.github/workflows/publish.yml

```
@@ -1,39 +1,159 @@
     - uses: actions/checkout@v4.2.2
       with:
+        ref: ${{ inputs.commit_hash }}
         submodules: 'recursive'
+    # Sunrise edit end
+
+    # Sunrise added start - подключаем приватное содержимое по настройке пресета
+    - name: Checkout private content
+      if: ${{ inputs.use_private_content || github.event_name == 'schedule' }}
+      uses: actions/checkout@v4.2.2
+      with:
+        repository: ${{ secrets.PRIVATE_REPOSITORY || secrets.SUNRISE_PRIVATE_REPO }}
```

actionlint (1.7.12)

[error] property "sunrise_private_repo" is not defined in object type {actions_runner_debug: string; actions_step_debug: string; announce_update_discord_role_id: string; announce_update_discord_webhook: string; changelog_discord_webhook: string; github_token: string; private_repository: string; private_repository_ssh_key: string; publish_token: string}

(expression)